src/terrain/services/metadata/tags.clj

```diff
@@ -16,7 +16,7 @@
 (defn- update-tags-targets
   [response]
   (letfn [(fmt-tgt ([tgt] (update tgt :id uuidify)))
-          (parse-resp ([response] (-> response :body slurp (json/parse-string true) :tags)))]
+          (parse-resp ([response] (-> response :body (json/parse-string true) :tags)))]
```

Comment:
Since `meta/update-attached-tags ` is already calling `:body` on its response, and it's making the request with `:as :json`, then I don't think the extra `:body` and `json/parse-string` calls are required here.

In fact, it looks like `parse-resp` is no longer needed at all, and `(:tags response)` can be used in the `deseq` below.